### PR TITLE
Switch to pangeo book theme

### PR DESCRIPTION
**Pull request by jbusecke**
- [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`

### CODE REVIEW

**Comment by rabernat** [comment]
Oh wow you beat me to it!

**Comment by jbusecke** [comment]
@rabernat @TomNicholas check this out: https://xgcm--482.org.readthedocs.build/en/482/

Looks pretty good to me. That version in the top left needs a tweak?

**Comment by TomNicholas** [comment]
Is it also supposed to have a greeny-blue pangeo-style bar across the top like the old docs theme did?

**Comment by jbusecke** [comment]
No I think that is the old styling. Check https://pangeo-forge.readthedocs.io/en/latest/

File: doc/conf.py [hunk 1]
```diff
@@ -173,15 +181,15 @@ def setup(app):
...
+#"<project> v<version> documentation".
+html_title = f"{project} v{version} documentation"
 
 # A shorter title for the navigation bar.  Default is the same as html_title.
 # html_short_title = None
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-# html_logo = None
+html_logo = _static/logo.png
```
Comment (TomNicholas) [inline doc/conf.py:192]:
```suggestion
html_logo = "_static/logo.png"
```
?

**Comment by jbusecke** [comment]
I made some tweaks to the logo. Its now on the top left sidebar and not so huge on the index page? What do you think?

**Pull request merged**